src/oceans/models/guidesK5.js

```diff
@@ -0,0 +1,485 @@
+        return state.yesCount + state.noCount >= 5;
+      },
+    },
+    style: "Info",
+    image: trashBottleImage,
+    imageStyle: imageStyleOverrides.bottle,
+  },
+  {
+    id: "fishvtrash-training-pause2",
+    textFn: () => I18n.t("fishvtrash-training-pause2"),
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Training,
+      fn: (state) => {
+        return state.yesCount + state.noCount >= 5;
+      },
+    },
+  },
+  {
+    id: "fishvtrash-training-pause3",
+    textFn: () => I18n.t("fishvtrash-training-pause3"),
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Training,
+      fn: (state) => {
+        return state.yesCount + state.noCount >= 15;
+      },
+    },
+    style: "Info",
+    image: trashCanImage,
+    imageStyle: imageStyleOverrides.can,
+  },
+  {
+    id: "fishvtrash-training-pause4",
+    textFn: () => I18n.t("fishvtrash-training-pause4"),
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Training,
+      fn: (state) => {
+        return state.yesCount + state.noCount >= 15;
+      },
+    },
+  },
+  {
+    id: "fishvtrash-training-pause5",
+    textFn: () => I18n.t("fishvtrash-training-pause5"),
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Training,
+      fn: (state) => {
+        return state.yesCount + state.noCount >= 30;
+      },
+    },
+  },
+  {
+    id: "fishvtrash-predicting-init1",
+    textFn: () => I18n.t("fishvtrash-predicting-init1"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Predicting },
+  },
+  {
+    id: "fishvtrash-predicting-init2",
+    textFn: () => I18n.t("fishvtrash-predicting-init2"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Predicting },
+  },
+  {
+    id: "fishvtrash-predicting-init3",
+    textFn: () => I18n.t("fishvtrash-predicting-init3"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Predicting },
+    noDimBackground: true,
+    arrow: "LowerRight",
+  },
+  {
+    id: "fishvtrash-pond-init1",
+    textFn: () => I18n.t("fishvtrash-pond-init1"),
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Pond,
+      fn: (state) => {
+        return state.fishData && state.totalPondFish !== null;
+      },
+    },
+  },
+  {
+    id: "fishvtrash-pond-recall",
+    textFn: () => I18n.t("fishvtrash-pond-recall"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Pond },
+    arrow: "UpperRight",
+  },
+  {
+    id: "fishvtrash-pond-init2",
+    textFn: () => I18n.t("fishvtrash-pond-init2"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Pond },
+    arrow: "LowerLeft",
+  },
+  {
+    id: "fishvtrash-pond-init3",
+    textFn: () => I18n.t("fishvtrash-pond-init3"),
+    when: { appMode: AppMode.FishVTrash, currentMode: Modes.Pond },
+    arrow: "LowerRight",
+  },
+  {
+    id: "creaturesvtrashdemo-predicting-init1",
+    textFn: () => 'This is a test string.',
```

Comment:
Here is an example unlocalized string that can be used to confirm which variant is showing.